_posts/2022-06-08-#22778.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: pr
+date: 2022-06-08
+title: "net processing: Reduce resource usage for inbound block-relay-only connections"
+pr: 22778
+authors: [jnewbery]
+components: ["p2p"]
+host: larryruane
+status: upcoming
+commit: 9db82f1bca0bb51c2180ca4a4ad63ba490e79da4
+---
+
+## Notes
+
+- (These notes describe behavior with or without this PR.)
+
+- We are a `bitcoind` node. Each of our peers is either inbound (our
```

Comment:
It's the Bitcoin Core PR Review Club, so there's no need to specify that our node is running bitcoind.
```suggestion
- Each of our node's peers is either inbound (our
```